lib/fake_braintree/dropin_assets.rb

```diff
@@ -0,0 +1,44 @@
+require 'fake_braintree/asset_versions'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.